Commands/List.js

```diff
@@ -1,7 +1,7 @@
 //this file will house all commands attached to the play command
 
-const list = ()=> {
-  console.log('List Command was called')
+const list = (message)=> {
+  message.channel.send('the current sounds in the directory are \n "cat" \n "fight" \n "replay"')
 }
 
 export default list
```

Comment:
Nitpick: this wasn't part of this PR, but I'd consider putting empty lines at the end of your files.